includes/Jobs/GenerateProductFeed.php

```diff
@@ -93,6 +93,10 @@ protected function process_items( array $items, array $args ) {
 		$processed_items = array();
 
 		foreach ( $products as $product ) {
+			// Check if product is enabled for synchronization.
+			if ( ! facebook_for_woocommerce()->get_product_sync_validator( $product )->passes_all_checks() ) {
```

Comment:
I suggest we add a filter here so external developers are able to filter as well which products to exclude

Reply:
That is a good idea. I will create an issue for this. I will not add it now because this requires more changes than just this line. We display if the product will be synchronized in other places so all of those spots will need to be compatible with what they are showing to the user. Maybe just adding a filter at the validator class will be the simplest way but that requires investigation beyond this PR.

Reply:
Issue created https://github.com/woocommerce/facebook-for-woocommerce/issues/2003